lib/libzdb/Makefile.am

```diff
@@ -0,0 +1,6 @@
+libzdb_la_CFLAGS = $(AM_CFLAGS) $(KERNEL_CFLAGS) $(LIBRARY_CFLAGS)
```

Comment:
It seems like we shouldn't need `$(KERNEL_CFLAGS)` here.

Let's also limit the visibility of these symbols since we don't intend for this to be a public interface.

```
libzdb_la_CFLAGS += -fvisibility=hidden
```

Reply:
That should be fixed now; if you agree, I'll just squash it.

Reply:
Looks good, I can squash this when merging.